railties/lib/rails/commands/credentials/credentials_command.rb

```diff
@@ -33,8 +34,18 @@ def edit
       def show
         require_application_and_environment!
 
-        say Rails.application.credentials.read.presence ||
-          "No credentials have been added yet. Use bin/rails credentials:edit to change that."
+        credentials = Rails.application.credentials.read.presence
+
+        if credentials
+          say credentials
+        elsif Rails.application.credentials.key.nil?
+          say(<<-end_of_message.squish)
+            Missing encryption key to decrypt secrets with.
+            Ask your team for your master key and put it in ENV["RAILS_MASTER_KEY"].
```

Comment:
What's the use of this? Won't `edit` need the same instructions? 😊

Reply:
This is for show messages that can understand when there is no key. 
(If this is not present, `No credentials have been added yet. Use bin/rails credentials:edit to change that."` is showed) 

`edit` is now unnecessary because master key creation is done whenever there is no key.

Reply:
This now blurs two error checks alongside the normal read flow. What do you think of joining those two like this?

```ruby
say Rails.application.credentials.read.presence || missing_credentials_explainer

def missing_credentials_explainer # Could use a better name.
  if Rails.application.credentials.key.nil?
    "Missing master key to decrypt credentials. See bin/rails credentials --help"
  else
    "No credentials…"
  end
end
```

Same would go for the encrypted command.

Reply:
:+1:  I fixed!